bot.md

```diff
@@ -0,0 +1,107 @@
+# Dev Instructions for Testing ZKMS Bot
+
+_Doing this steps in order is more likely to result in success_
+
+### 0. Prerequisites
+
+- Install [Telegram Desktop](https://desktop.telegram.org/)
+- In the Telegram app, [Enable Web View Debug](https://core.telegram.org/bots/webapps#debug-mode-for-mini-apps)
+- Obtain a Telegram Bot [Token](https://core.telegram.org/bots/tutorial#obtain-your-bot-token)
+- Get a Pretix API token from a member of the 0xPARC PCD team.
+
+### 1. Set up environment
+
+- If you haven't set up the repository before, copy over the example environment file in `passport-server`.
+
+```sh
+cp apps/passport-server/.env.local.example apps/passport-server/.env
```

Comment:
if the developers on this team follow these instructions, they will mess up their config - maybe more granular instructions would be better here

Reply:
`If you haven't set up the repository before` I guess this is not clear enough. Also probably unnecessary since we already mention this in the root README.md